src/stores/Widgets/CatalogWidget/CatalogWidgetStore.ts

```diff
@@ -97,10 +108,14 @@ export class CatalogWidgetStore {
     @observable sizeAxisTabId: CatalogSettingsTabs.SIZE_MINOR | CatalogSettingsTabs.SIZE_MAJOR;
     @observable sizeColumnMinLocked: boolean;
     @observable sizeColumnMaxLocked: boolean;
+    @observable sizeUnit: AngularSizeUnit | string;
+    @observable angularSizeUnit: AngularSizeUnit;
```

Comment:
It might be helpful if we mention somewhere (naming or TsDoc) that these two variables are for completely different modes (sizeUnit for CANVAS and angularSizeUnit for WOLRD).